dojo/finding/views.py

```diff
@@ -1542,143 +1981,169 @@ def apply_cwe_mitigation(apply_to_findings, template, update=True):
+                    " and "
+                    + str(count)
+                    + " "
+                    + pluralize(count, "finding,findings")
+                    + " "
+                )
             else:
                 apply_message = ""
 
             messages.add_message(
                 request,
                 messages.SUCCESS,
-                'Template ' + apply_message + 'updated successfully.',
-                extra_tags='alert-success')
-            return HttpResponseRedirect(reverse('templates'))
+                "Template " + apply_message + "updated successfully.",
+                extra_tags="alert-success",
+            )
+            return HttpResponseRedirect(reverse("templates"))
         else:
             messages.add_message(
                 request,
                 messages.ERROR,
-                'Template form has error, please revise and try again.',
-                extra_tags='alert-danger')
+                "Template form has error, please revise and try again.",
+                extra_tags="alert-danger",
+            )
 
     count = apply_cwe_mitigation(True, template, False)
     add_breadcrumb(title="Edit Template", top_level=False, request=request)
-    return render(request, 'dojo/add_template.html', {
-        'form': form,
-        'count': count,
-        'name': 'Edit Template',
-        'template': template,
-    })
+    return render(
+        request,
+        "dojo/add_template.html",
+        {
+            "form": form,
+            "count": count,
+            "name": "Edit Template",
+            "template": template,
+        },
+    )
 
 
 @user_has_global_permission(Permissions.Finding_Delete)
 def delete_template(request, tid):
     template = get_object_or_404(Finding_Template, id=tid)
-
-    form = DeleteFindingTemplateForm(instance=template)
-
-    if request.method == 'POST':
+    if request.method == "POST":
         form = DeleteFindingTemplateForm(request.POST, instance=template)
         if form.is_valid():
             template.delete()
             messages.add_message(
                 request,
                 messages.SUCCESS,
-                'Finding Template deleted successfully.',
-                extra_tags='alert-success')
-            return HttpResponseRedirect(reverse('templates'))
+                "Finding Template deleted successfully.",
+                extra_tags="alert-success",
+            )
+            return HttpResponseRedirect(reverse("templates"))
         else:
             messages.add_message(
                 request,
                 messages.ERROR,
-                'Unable to delete Template, please revise and try again.',
-                extra_tags='alert-danger')
+                "Unable to delete Template, please revise and try again.",
+                extra_tags="alert-danger",
+            )
     else:
         return HttpResponseForbidden()
 
 
-def download_finding_pic(request, token):
+def download_finding_pic(token):
```

Comment:
Okay confirmed that the request object does need to be present because this function is connected to a URL path. Please put the request arg back

https://github.com/DefectDojo/django-DefectDojo/blob/e76dc86e2ce1a53ac00ff46ccdd0e758337ccd4d/dojo/finding/urls.py#L86

Reply:
OK

Reply:
https://github.com/bancolombia/django-DefectDojo/blob/a3c09694810b56ca30eb2db10e52be172783c3ed/dojo/finding/views.py#L2111